script.js

```diff
@@ -0,0 +1,8 @@
+menu.onclick = function myFunction() {
+  const x = document.getElementById('myTopnav');
```

Comment:
Not sure that "x" is great name for a variable. Try to find somthing more readable.

You can read about variables naming here:
https://spin.atomicobject.com/2017/11/01/good-variable-names/
or here:
http://wiki.c2.com/?GoodVariableNames or somewhere else.